### DISCUSSION TITLE
Best Option: Custom Field? RSVP?

### DISCUSSION THREAD

**Discussion by AllenFreeman** [liedekef/events-made-easy#363] [Q&A]
The site I'm working on runs golf tournaments. Players are allowed to sign up for an event starting on a specific day/time and ending on a specific day/time. (ex: for a May 1 tournament, signups open on April 15 at 7am and close on April 25 at 11:59pm.) But the start/end dates for signups aren't consistent across all events, so I have to list specific times/dates in each event.

The goal: I'd like to post a list all of the events that are currently open for sign-up on the homepage using the eme_events option.

What's the best way to do this? Create custom fields for Registration Opens/Registration Closes? Use your RSVP system?

Thank you for any ideas and tips to execute.

**Comment by liedekef** [comment]
Each event has RSVP start/end times as well, using [eme_if] and checking the condition #_IS_RSVP_STARTED or #_IS_RSVP_ENDED you can filter out events in a list so only those actually open for registration are showing.

**Reply by AllenFreeman** [reply]
Thank you for the reply. I found your page https://www.e-dynamics.be/wordpress/conditional-tags/ and tried to create a condition check, but couldn't find an example close enough to figure it out on my own. Assuming an RSVP start and end date are set for an event, what would the eme_if string be to check the condition to display only the events that are still open for registration?

**Reply by liedekef** [reply]
`[eme_if tag='#_IS_RSVP_STARTED' value='1'] .... [/eme_if]` should do the trick
This should be used in the [eme_events] template_id option when showing events
